pkg/util/util.go

```diff
@@ -0,0 +1,134 @@
+package util
+
+func SliceContains[T comparable](slice []T, target T) bool {
+	for _, item := range slice {
+		if item == target {
+			return true
+		}
+	}
+	return false
+}
+
+func MapSlice[T any, TResult any](slice []T, mapper func(item T) TResult) []TResult {
```

Comment:
This is awkward. Every language other than C# calls this operation "map", however golang calls dictionaries "map" as well, so there's a confusion of terms.

Maybe we should rename it to `SliceSelect` to match C# or something? Perhaps `TransformSlice` to align with `std::transform` in C++ instead?. Or we could just leave it 🤷

Reply:
Let’s use SliceTransform

Reply:
Done :-)